### PR TITLE
Remove release-notes

### PR DESCRIPTION
**Pull request by joshaber**
Since it’s been reporting the wrong release notes, it’s doing more harm than good :disappointed: 

We’ll link to the release notes from About now.

/cc @atom/feedback

### CODE REVIEW

**Comment by joshaber** [comment]
I'll back port this onto `stable` once it's merged.

File: package.json [hunk 1]
```diff
@@ -102,7 +102,6 @@
     "notifications": "0.62.1",
     "open-on-github": "0.40.0",
     "package-generator": "0.41.0",
-    "release-notes": "0.53.0",
```
Comment (kevinsawicki) [inline package.json]: :fire: :green_heart:

**Comment by 50Wliu** [comment]
Refs (and should close) #10055.

**Comment by mnquintana** [comment]
Is this going to change the update flow at all? (Like, will there now be no status bar squirrel indicating a new version's available / been installed, for instance?)

**Comment by benogle** [comment]
> Is this going to change the update flow at all? (Like, will there now be no status bar squirrel indicating a new version's available / been installed, for instance?)

This. I think the status bar squirrel is done in the release notes package, but that logic could move to the about package.

**Comment by lee-dohm** [comment]
Also should close https://github.com/atom/atom/issues/5692

**Comment by joshaber** [comment]
>I think the status bar squirrel is done in the release notes package, but that logic could move to the about package.

Or perhaps the `status-bar` package itself?

**Comment by lee-dohm** [comment]
> Or perhaps the status-bar package itself?

:+1: I like this idea.

**Comment by simurai** [comment]
There is also https://github.com/atom/about/pull/5, so the about package would fit too in case that direction will move forward.

**Comment by joshaber** [comment]
Ah, good point @simurai. I'll go with `about` then :+1:

**Comment by joshaber** [comment]
Once this goes green I'll merge and cherry-pick it into stable.

**Comment by benogle** [comment]
Cool :+1: 

> cherry-pick it into stable

beta?

**Comment by joshaber** [comment]
>>cherry-pick it into stable

>beta?

Yeah or that. Basically the same since they'll both be rolled into 1.4.0.

**Comment by joshaber** [comment]
Looks like the about specs are hanging on janky for some reason.

**Comment by joshaber** [comment]
Well, it passed on janky-osx4 and Travis so it seems to be a problem specific to janky-osx2.

**Comment by joshaber** [comment]
Cherry picked into `beta`: https://github.com/atom/atom/commit/d229fed7e73c0f89631ed3b6c2cf16497d0ccbab

**Pull request merged**